internal/api/oidc/token_jwt_profile.go

```diff
@@ -42,7 +42,7 @@ func (s *Server) JWTProfile(ctx context.Context, r *op.Request[oidc.JWTProfileGr
 	session, err := s.command.CreateOIDCSession(ctx,
 		user.ID,
 		user.ResourceOwner,
-		jwtReq.Subject,
+		"",
 		scope,
 		domain.AddAudScopeToAudience(ctx, nil, r.Data.Scope),
 		nil,
```

Comment:
i guess we need to add an additional `UserAuthMethodTypeJWT` / `UserAuthMethodTypePrivateKey` and pass it here as the auth methods, otherwise the token will not be valid for a ZITADEL call

and also just noticed the token reason below should be jwt proflle instead of client credentials